src/Microsoft.EntityFrameworkCore.Tools/tools/install.ps1

```
@@ -1,5 +1,9 @@
 ﻿param ($installPath, $toolsPath, $package, $project)
 
+if ($PSVersionTable.PSVersion.Major -lt 3) {
+    throw "EF commands do not support PowerShell version $($PSVersionTable.PSVersion). Please upgrade PowerShell to 3.0 or greater and restart Visual Studio."
+}
+
```

It seems NuGet eats errors in init.ps1 when installing a package.